recipes/acdc/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "acdc" %}
+{% set version = "1.02" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/mlux86/acdc/archive/refs/heads/master.zip"
```

Comment:
The GitHub repo doesn't have any version tags. Could this be changed to point to the specific commit for this version bump instead of the head of the master branch?

Reply:
I have changed the source url to point to the latest commit, please review.